src/libServer/JSONConversion.cpp

```diff
@@ -0,0 +1,129 @@
+/**
+* Copyright (c) 2018 Zilliqa 
+* This source code is being disclosed to you solely for the purpose of your participation in 
+* testing Zilliqa. You may view, compile and run the code for that purpose and pursuant to 
+* the protocols and algorithms that are programmed into, and intended by, the code. You may 
+* not do anything else with the code without express permission from Zilliqa Research Pte. Ltd., 
+* including modifying or publishing the code (or any part of it), and developing or forming 
+* another public or private blockchain network. This source code is provided ‘as is’ and no 
+* warranties are given as to title or non-infringement, merchantability or fitness for purpose 
+* and, to the extent permitted by law, all liability for your use of the code is disclaimed. 
+* Some programs in this code are governed by the GNU General Public License v3.0 (available at 
+* https://www.gnu.org/licenses/gpl-3.0.en.html) (‘GPLv3’). The programs that are governed by 
+* GPLv3.0 are those programs that are located in the folders src/depends and tests/depends 
+* and which include a reference to GPLv3 in their program files.
+**/
+
+
+#include <array>
+#include <boost/multiprecision/cpp_int.hpp>
+#include <string>
+#include <vector>
+
+#include "libCrypto/Schnorr.h"
+#include "libData/AccountData/Address.h"
+#include "libData/AccountData/Transaction.h"
+#include "libData/BlockChainData/DSBlockChain.h"
+#include "libData/BlockChainData/TxBlockChain.h"
+#include "libData/BlockData/Block.h"
+#include "JSONConversion.h"
+
+
+
+using namespace std;
+using namespace boost::multiprecision;
+
+
+const Json::Value JSONConversion::convertBoolArraytoJson(vector<bool> v)
```

Comment:
It's better to use const reference here to pass the parameter (e.g., `const vector<bool> &v`)

Reply:
Okay I will do that

Reply:
How about txblock and dsblock? @KaustubhShamshery